README.md

```diff
@@ -294,27 +298,37 @@ or
 
 ...etc.
 
-### Hiding Vulkan API Symbols
 
-You can optionally build **MoltenVK** with the Vulkan API static call symbols (`vk*`) hidden, to avoid 
+<a name="hiding_vulkan_symbols"></a>
+### Hiding _Vulkan_ API Symbols
+
+You can optionally build **MoltenVK** with the _Vulkan_ API static call symbols (`vk*`) hidden, to avoid 
 library linking conflicts when statically bound to a _Vulkan_ loader that also exports identical symbols.
 
 To do so, when building **MoltenVK**, set the build setting `MVK_HIDE_VULKAN_SYMBOLS=1`.
-This build setting can be set in the `MoltenVK.xcodeproj` *Xcode* project,
-or it can be included in any of the `make` build commands. For example:
-
-	make MVK_HIDE_VULKAN_SYMBOLS=1
-or
-
-	make macos MVK_HIDE_VULKAN_SYMBOLS=1
-
-...etc.
+This build setting can be set in the `MoltenVK.xcodeproj` *Xcode* project, or it can be 
+included in any of the `make` command-line build commands [mentioned above](#command_line_build).
 
 With `MVK_HIDE_VULKAN_SYMBOLS=1`, the _Vulkan_ `vkGetInstanceProcAddr()` call remains
 statically bound, to provide the application with the ability to retrieve the remaining
 _Vulkan_ API calls as function pointers.
 
 
+<a name="metal_private_api"></a>
+### Accessing _Metal_ Private API calls
+
+You can optionally build **MoltenVK** with access to _Metal_ private API calls. 
+Doing so will allow **MoltenVK** to extend functionality by using certain private _Metal_ API calls, 
+however, doing so will disqualify the app from being distributed via _Apple_ App Stores.
```

Comment:
I somehow really dislike how `_Metal_ private API calls` sounds and the repetition of the `doing so`. I feel like this reads much better:
> You can optionally build **MoltenVK** with access to private _Metal_ API calls.
Doing so will allow **MoltenVK** to extend its functionality by using certain private _Metal_ API calls, but it will also disqualify the app from being distributed via _Apple_ App Stores.

Reply:
Thanks. I struggled a bit with wording there. Fixed now.